web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
> What do you say about leaving the true/false cancel expectation as an argument, but moving all the ShareData expectations into a dictionary?

re: true/false, it's just that I've seen humans be the biggest bottleneck to getting compat issues fixed. We should be minimizing the amount of work anyone who hits a fail needs to do in order to work out what has gone wrong (see also @tobie's comment about having to look up the function signature).  

@sideshowbarker, you've been at this for a few years, would you agree?